### PR TITLE
zipkin-anorm: create an AnormDB SpanStore

### PR DESCRIPTION
**Pull request by sprsquish**
SpanStoreValidator has been updated to provide more helpful messages.

### CODE REVIEW

File: zipkin-anormdb/src/test/scala/com/twitter/zipkin/storage/anormdb/AnormSpanStoreTest.scala [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2012 Twitter Inc.
```
Comment (nshkrob) [inline zipkin-anormdb/src/test/scala/com/twitter/zipkin/storage/anormdb/AnormSpanStoreTest.scala]: 2014? Also add the header to other files.

File: zipkin-anormdb/src/main/scala/com/twitter/zipkin/storage/anormdb/SpanStoreDB.scala [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+  /**
+   * Execute SQL in a transaction.
+   *
+   * Example usage:
+   *
+   * db.withTransaction(conn, { implicit conn: Connection =>
+   *   // Do database updates
+   * })
+   */
+  def withTransaction[A](conn: Connection, code: Connection => A): Try[A] = {
```
Comment (nshkrob) [inline zipkin-anormdb/src/main/scala/com/twitter/zipkin/storage/anormdb/SpanStoreDB.scala]: This is not used anywhere.

File: zipkin-anormdb/src/main/scala/com/twitter/zipkin/storage/anormdb/SpanStoreDB.scala [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+        |  span_id BIGINT NOT NULL,
+        |  parent_id BIGINT,
+        |  trace_id BIGINT NOT NULL,
+        |  span_name VARCHAR(255) NOT NULL,
+        |  debug SMALLINT NOT NULL,
+        |  duration BIGINT,
+        |  created_ts BIGINT
+        |)
+      """.stripMargin).execute()
+    //SQL("CREATE INDEX trace_id ON zipkin_spans (trace_id)").execute()
```
Comment (nshkrob) [inline zipkin-anormdb/src/main/scala/com/twitter/zipkin/storage/anormdb/SpanStoreDB.scala]: Is this needed? Same for 'create index' statements below.

File: zipkin-anormdb/src/test/scala/com/twitter/zipkin/storage/anormdb/AnormSpanStoreTest.scala (hunk 1 above)
Comment (sprsquish) [inline zipkin-anormdb/src/test/scala/com/twitter/zipkin/storage/anormdb/AnormSpanStoreTest.scala]: fixed

File: zipkin-anormdb/src/main/scala/com/twitter/zipkin/storage/anormdb/SpanStoreDB.scala (hunk 2 above)
Comment (sprsquish) [inline zipkin-anormdb/src/main/scala/com/twitter/zipkin/storage/anormdb/SpanStoreDB.scala]: copy/paste holdover. removed

File: zipkin-anormdb/src/main/scala/com/twitter/zipkin/storage/anormdb/SpanStoreDB.scala (hunk 3 above)
Comment (sprsquish) [inline zipkin-anormdb/src/main/scala/com/twitter/zipkin/storage/anormdb/SpanStoreDB.scala]: copy/paste holdover. removed

**Comment by nshkrob** [comment]
Ship it!

**Pull request closed without merging**